### ISSUE TITLE
Junit dependency scope

### ISSUE THREAD

**Issue by jmini** [RepreZen/KaiZen-OpenApi-Parser#159]
Version:

```
<dependency>
    <groupId>com.reprezen.kaizen</groupId>
    <artifactId>openapi-parser</artifactId>
    <version>2.1.0.201803270245</version>
</dependency>
```

The `junit` dependency has `compile` scope.

Why isn't the scope `test`?

---

I think there is a similar problem in the upstream dependency: `com.reprezen.jsonoverlay:jsonoverlay:2.0.0.201803231630`

**Comment by andylowry** [comment]
Yes, thanks @jmini. I'll fix that in both.

**Comment by andylowry** [comment]
Fixed in #160

**Comment by jmini** [comment]
Verified in the latest jsonoverlay release:

```xml
	<groupId>com.reprezen.jsonoverlay</groupId>
	<artifactId>jsonoverlay</artifactId>
	<version>2.0.2-201805111136</version>
```

http://central.maven.org/maven2/com/reprezen/jsonoverlay/jsonoverlay/2.0.2-201805111136/jsonoverlay-2.0.2-201805111136.pom

---

I will verify it in the next KaiZen-OpenApi-Parser release (no hurry).
